### DISCUSSION TITLE
Does this work per user on the server?

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [Taxel/PlexTraktSync#680] [Q&A] [answered]
If I have 6 users on the Plex server, can this independently sync to 6 different Trakt accounts? Or would it mix everyone's metadata together?

Thank you.

**Comment by mvanbaak** [comment, marked as answer]
I created a configuration per user, and run PTS 6 times, once for each user.
Each plex user has their own trakt account. This keeps things easy to understand.

See https://github.com/Taxel/PlexTraktSync/issues/629

**Comment by ksya** [comment]
Are managed users (multiple users on one account) in Plex supported? I have run it once but did not get the option to choose the user. It synced the main user, which was fine in this case, but I want to sync another user under my account as well.

**Reply by WoodpeckerBaby** [reply]
I think it only works with full Plex Accounts, which requires Plex Pass or be invited as friends, which you would lose fast user switching. 

So, I ditched this project and moved to Jellyfin.

**Reply by glensc** [reply]
PlexTraktSync supports Plex Media Server managed users if you log in with Plex Media Server admin account (owner). Use `plex-login` subcommand to log in with a different user.

Theoretically (I have not tested myself) if you log in with the managed user itself, then you can't pick managed user as you are already the managed user.

Also, don't mix Plex Media Server and PlexTraktSync, they are two different products. PlexTraktSync is not affiliated with Plex Media Server in any way and its code is all open source.

**Reply by WoodpeckerBaby** [reply]
> Also, don't mix Plex Media Server and PlexTraktSync, they are two different products. PlexTraktSync is not affiliated with Plex Media Server in any way and its code is all open source.

I'm aware of the difference. I meant the whole ecosystem is frustrating, playing me like cats chasing tails. Jellyfin uses a much newer technology stack, with a very friendly community, and things just work.

**Reply by glensc** [reply]
You may be all super knowing, but don't confuse other users.

**Reply by ksya** [reply]
> 

Thanks! Unfortunately it does not ask me for the managed user. I see in the code that it should ask "Select the user you would like to use:". This is what I use: sudo docker-compose run --rm plextraktsync plex-login

**Reply by glensc** [reply]
How you invoke plextraktsync is irrelevant, what matters is what credentials you fill. if it did not ask for you managed user, you did not login with an account privileged enough to see managed users:
- https://github.com/Taxel/PlexTraktSync/blob/961fe7b3c35458f0f543c8c2cfa15cdae64b187e/plextraktsync/commands/plex_login.py#L171-L175

**Reply by ksya** [reply]
I do login with the user who created the users (me). I don't even have a Plex Pass and cannot add other users with an account. Now it only syncs my main account, but does not give me options for choosing a managed user.

**Reply by ksya** [reply]
I have found the problem. The code checks on friend users, but omits the managed users this way:
line 45:     users = [u.title for u in account.users() if u.friend]

The "if u.friend" part needs to be removed. I will make a pull request

**Reply by glensc** [reply]
https://github.com/Taxel/PlexTraktSync/pull/694
